enterprise/server/raft/cache/cache.go

```diff
@@ -72,6 +81,10 @@ type Config struct {
 	PartitionMappings []disk.PartitionMapping
 }
 
+type accessTimeUpdate struct {
```

Comment:
nit: comment that this struct exists to put in the accesses chan

Reply:
done